### DISCUSSION TITLE
feature request: role icons

### DISCUSSION THREAD

**Discussion by Vallmerka** [stoatchat/discussions#999] [Feature Suggestions]
### What do you want to see?

I would like to be able to add icons (.png) to roles in the server.
In general, it will be a very pleasant feature.

And how about increasing the role extensions

**Comment by DaarkDev** [comment]
Yes please, I would really love to have this!

**Comment by geox25** [comment]
would be ideal, so for example I could give certain roles a verified icon/badge, or something else. would also really like animated ones, so anything from png, jpg, and gifs.

**Comment by ghost** [comment]
+1 

Would be a really cool feature to have at some point in the future, maybe in new frontend??
